src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import core.mate.academy.model.Machine;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Machine> {

```

Comment:
ExcavatorProducer returns Machine? Or maybe he should return some specific machine